ToolBox/README.md

```diff
@@ -7,34 +7,65 @@ Lean Data ToolBox
 
 ## Introduction ##
 
-Lean Engine is an open-source fully managed C# algorithmic trading engine built for desktop and cloud usage. It was designed in Mono and operates in Windows and Linux.
-
-The following is a collection of helper projects for data download and storage to Lean.
-
-## QuantQuote Converter ##
-
-The QuantQuote converter ports an extracted QuantQuote order to QuantConnect data format and saves it to your LEAN directory. The user must enter three key parts of information:
-
- - Directory where your QuantQuote order is extracted.
- - Directory where Lean Data is located "Lean/Data".
- - Resolution of the QuantQuote data.
-
-## Oanda Downloader ##
-
-Download data directly from the Oanda database using your personal access token. The downloader will save the information to your FX directory.
-
-## Dukascopy Downloader ##
-
-Download data from Dukascopy website and convert it into LEAN engine format. Save the data to your personal LEAN data directory.
-
-## Yahoo Downloader ##
-
-Download data from Yahoo into daily files and store them in your local data directory.
-
-## Google Downloader ##
-
-Download data from Google and store them in your local data directory.
-
-  [1]: https://lean.quantconnect.com "Lean Open Source Home Page"
-  [2]: https://lean.quantconnect.com/docs "Lean Documentation"
-  [3]: https://github.com/QuantConnect/Lean/archive/master.zip
+Lean Engine is an open-source algorithmic trading engine built for easy strategy research, backtesting and live trading. We integrate with common data providers and brokerages so you can quickly deploy algorithmic trading strategies.
+
+The ToolBox project is a command line program which wrapps over 15 tools.
+
+## Usage
+
+Each tool requires a different set of parameters, the only **obligatory option is '--app='**, which defines the target tool and is case insensitive.
```

Comment:
nit; for API's normal term is "required". obligatory option -> required argument.